hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/protocolPB/OzoneManagerProtocolServerSideTranslatorPB.java

```diff
@@ -296,7 +320,10 @@ private OMResponse submitRequestDirectlyToOM(OMRequest request) {
       ExitUtils.terminate(1, errorMessage, ex, LOG);
       Thread.currentThread().interrupt();
     }
-    return omClientResponse.getOMResponse();
+    OMLockDetails omLockDetails = omClientResponse.getOmLockDetails();
```

Comment:
The handleReadRequest responses are not checked in both submitRequestDirectlyToOM and submitReadRequestToOM.

Reply:
I see. Can we extract the "call.getProcessingDetails().add" logic to an individual function, can call it directly here? to avoid generate a new OMResponse to set  omLockDetails.toProtobufBuilder().

Reply:
Just realized non-ratis's processing time has already been set in "https://github.com/symious/ozone/blob/HDDS-8974/hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/lock/OzoneManagerLock.java#L616", so removed the logics in `submitRequestDirectlyToOM`. PTAL.